odt2txt/PKGBUILD

```diff
@@ -0,0 +1,44 @@
+# Maintainer: Alberto Fanjul <albertofanjul@gmail.com>
+
+source=(https://downloads.sourceforge.net/sourceforge/${pkgname}/${pkgname}-${pkgver}.tgz)
+sha256sums=('b297752910a404c1435e703d5aedb4571222bd759fa316c86ad8c8bbe58c6d1b')
+depends=('perl' 'unzip')
```

Comment:
I check again original PKGBUILD and it do not have that dependency

https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/odt2txt

Will remove it